src/models/TerraformComponentAttribute.js

```diff
@@ -22,6 +22,14 @@ class TerraformComponentAttribute extends ComponentAttribute {
      */
     this.isDynamic = props.isDynamic || false;
   }
+
+  /**
+   * Check if the attribute is a variable.
+   * @returns {boolean} - true if the attribute is a variable otherwise false.
+   */
+  get isVariable() {
```

Comment:
merge this commit with TerraformComponentAttribute.

Reply:
There is only one commit that modify this file, so no changes.